spec/salestation/app/errors/dependency_currently_unavailable_spec.rb

```diff
@@ -8,4 +8,16 @@
   it 'has message' do
     expect(create_dependency_currently_unavailable.message).to eq(message)
   end
+
+  context 'when debug_message is not required' do
```

Comment:
I don't understand the description of the `context`. I don't see anything in the context `subject` nor `let`s that communicates `debug_message` being not required to me.